services/discord/discord-online-count.service.js

```diff
@@ -0,0 +1,76 @@
+import Joi from 'joi'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  approximate_presence_count: nonNegativeInteger,
+  approximate_member_count: nonNegativeInteger,
+}).required()
+
+const documentation = `
+<p>
+  The Discord badge requires the <code>SERVER INVITE CODE</code> in order access the Discord JSON API, this invite code should be set to never expire.
+</p>
+<p>
+  The <code>SERVER INVITE CODE</code> can be located at the end of the invite url.
+</p>
+`
+
+export default class DiscordOnlineCount extends BaseJsonService {
+  static category = 'chat'
+
+  static route = {
+    base: 'discord/online-count',
```

Comment:
Only thoughts on possible friendly names all relay the same sort of meaning
`online-total` - same as `online`
`online-count` - same as `online`
`online-with-total` - is probably fine, it's not too long IMO?

Reply:
Recognize this option might add some internal complexity within the code, but from an end user perspective I wonder if it would be preferable to have the "with total" option provided as a boolean query parameter?

Not sure if that would be too inconsistent with existing badges, but seems like it would be an easier way for a user to build the badge from the frontend site.

Otherwise, I'd vote `online-with-total`